onnxscript/tests/function_libs/torch_aten/extra_opinfo.py

```diff
@@ -201,6 +202,95 @@ def sample_inputs_layer_norm(op_info, device, dtype, requires_grad, **kwargs):
         )
 
 
+class _TestParamsMaxPoolEmptyStrideBase:
```

Comment:
Why do we keep a local custom test data here?

Reply:
I just copied the class and edited it to contain empty strides. Any suggestions?